paseos/attitude/geometric_model.py

```diff
@@ -0,0 +1,123 @@
+from loguru import logger
+import numpy as np
+import trimesh
+
+
+class CuboidGeometricModel:
```

Comment:
would suggest to just make on GeometricModel , a cuboid is just a specific case of a mesh

Reply:
Will do